crates/storage/errors/src/db.rs

```diff
@@ -1,61 +1,64 @@
-#[cfg(feature = "std")]
-use std::{fmt::Display, str::FromStr, string::String};
+// #[cfg(feature = "std")]
+// use std::{fmt::Display, str::FromStr, string::String};
 
-#[cfg(not(feature = "std"))]
+// #[cfg(not(feature = "std"))]
 use alloc::{
     boxed::Box,
     format,
     string::{String, ToString},
     vec::Vec,
 };
 
-#[cfg(not(feature = "std"))]
-use core::{fmt::Display, str::FromStr};
+// #[cfg(not(feature = "std"))]
+use core::{fmt, fmt::{Display, Debug}, str::FromStr};
 
 /// Database error type.
-#[derive(Clone, Debug, PartialEq, Eq, thiserror_no_std::Error)]
+#[derive(Clone, Debug, PartialEq, Eq, derive_more::Display)]
 pub enum DatabaseError {
     /// Failed to open the database.
-    #[error("failed to open the database: {0}")]
+    #[display(fmt="failed to open the database: {_0}")]
```

Comment:
fmt nit for every occurrence
```suggestion
    #[display(fmt = "failed to open the database: {_0}")]
```

Reply:
done